docs/pyenv-scl.md

```diff
@@ -0,0 +1,121 @@
+# Kapitan on Older Operating Systems
+
+## Introduction
+
+Kapitan requires Python 3.6, and you need to be able to install the dependencies in the 
+[requirements file](https://github.com/deepmind/kapitan/blob/master/requirements.txt) file.  However, sometimes this isn't entirely straightforward and you may not be able or willing to install new versions of Python system-wide.  
+
+We do provide a [dockerfile](dockerfile) which you can use to run Kapitan in a container, but if this isn't practical or possible either, you may wish to use one of the following options:
+* [PyEnv](https://github.com/pyenv/pyenv) (Linux, distro-agnostic)
+* [Software Collections](https://www.softwarecollections.org)(RHEL-based distros)
+
+Both of these projects allow you to use a different version of Python specifically for your work with or on Kapitan.  They work similarly to [Python Virtual Environments](https://docs.python.org/3/tutorial/venv.html) but with more isolation from the lower-level OS-wide Python installation.  Both of these projects manipulate your shell environment variables to make sure you're using the right binaries and modules.  This document assumes you're using the bash shell. 
+
+PyEnv and Software Collections are Google projects, so please exercise your own judgment as to whether these projects are suitable for your circumstances.
+
+## On Debian-based Operating Systems - PyEnv
+
+Here at Streams, we use a Debian-based operating system for our day-to-day work.  We've found that [PyEnv](https://github.com/pyenv/pyenv) works well for us.
+### Getting Started
+Take a look at the [PyEnv](https://github.com/pyenv/pyenv) project on Github.  There are two options for installing this project.  The author makes a separate installer script available at [https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer](this Github link) or you can manually install it.  We recommend using the automated installer unless you have reason to use the manual installation process.
+
+### The Automated Installer
+
+To use the installer, we would recommend downloading the installer script and examining it before you execute it.  
+```bash
+$ curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer > pyenv-installer
+```
+Then, once you've checked it, you can execute it.  PyEnv doesn't need root privileges, so you can run it without using sudo
+```bash
```

Comment:
If it's a console command, "console" produces a bit more readable output than bash in these code blocks. 
https://stackoverflow.com/questions/20303826/highlight-bash-shell-code-in-markdown 
Also I'd recommend a newline before and after each codeblock to improve readability when reading the source code. :+1:

Reply:
Didn't know about that option. Thanks